mabwiser/mab.py

```diff
@@ -969,6 +969,18 @@ def neighborhood_policy(self):
         else:
             return None
 
+    @property
+    def cold_arms(self) -> List[Arm]:
+        if not self.neighborhood_policy:
+            # No neighborhood policy, cold arms are calculated at the learning policy level
+            return self._imp.cold_arms
+
+        else:
```

Comment:
Design Q: if not defined, should return empty List() or Abort with some exception? No inclination one way or the other..

Reply:
I think we are de-facto saying that NPs do not have any cold arms so returning empty list makes sense to me.